### PR TITLE
[WIP] Make Claim modified event cancellable

### PR DESCRIPTION
**Pull request by 112RG**
Allow the event to be cancelled on request of #383

### CODE REVIEW

File: src/main/java/me/ryanhamshire/GriefPrevention/events/ClaimModifiedEvent.java [hunk 1]
```diff
@@ -11,14 +12,16 @@
  * a claim has changed. The CommandSender can be null in the event that the modification is called by the plugin itself.
  * Created by Narimm on 5/08/2018.
  */
-public class ClaimModifiedEvent extends Event {
+//Allow the event to be cancelled //112madgamer
```
Comment (RoboMWM) [inline src/main/java/me/ryanhamshire/GriefPrevention/events/ClaimModifiedEvent.java]: Unnecessary comment, and in the wrong place anyways.

Please remove.

**Review by RoboMWM — DISMISSED** [review]
Looks like good progress otherwise. Still needs to act upon this cancel status.

**Review by RoboMWM — CHANGES_REQUESTED** [review]
Looks good, but the plugin needs to check cancel status and cancel modification if this event is canceled. Idk how easy that'll be.

**Comment by Jikoo** [comment]
Implemented by #942

**Pull request closed without merging**